addons/wowl/static/src/views/graph/graph_model.ts

```diff
@@ -14,6 +13,28 @@ const NO_CONTENT_HELPER_DESCRIPTION = _lt(
   "Pie chart cannot mix positive and negative numbers. Try to change your domain to only display positive results"
 );
 
+export interface GraphModelParams {
+  modelName: string;
+  fields: Fields;
+}
+
+export interface GraphModelLoadParams {
```

Comment:
We end up with the same thing as before ModelParams and LoadParams,... But we could move the ModelParams to LoadParams